### DISCUSSION TITLE
Rearrange PHP versions in the "Supported PHP Extensions" section

### DISCUSSION THREAD

**Discussion by shinsenter** [mlocati/docker-php-extension-installer#948] [Ideas]
I think the "[Supported PHP Extensions](https://github.com/mlocati/docker-php-extension-installer?tab=readme-ov-file#supported-php-extensions)" section would be easier to view if the PHP versions were listed from the newest to the oldest. This would help users quickly find compatibility information for the most current PHP versions.

<img width="890" alt="Screenshot 2024-07-30 at 18 01 23" src="https://github.com/user-attachments/assets/a0c372b2-56d9-446c-a99d-6aea84976d9f">

Could we update the order to reflect this suggestion?

Thank you!

**Comment by mlocati** [comment]
Done in #949
